source/User_Guide/Transactional_Templates/how_to_send_an_email_with_dynamic_transactional_templates.md

```diff
@@ -0,0 +1,116 @@
+1. To begin editing your new template, click **Add Version**.
+1. Select an editor and click **Continue**.
+1. Design your template. For more information on designing a Dynamic Transactional Template using the Design Editor or Code Editor, see [Create and edit Dynamic Transactional Templates]({{root_url}}/User_Guide/Transactional_Templates/create_and_edit_dynamic_transactional_templates.html).
+
+Here's a sample template: <a href="/source/assets/sample_dynamic_template.html" target="_blank">Hipster Cat Cafe</a>
+
+{% anchor h2 %}
+Send a transactional email
+{% endanchor %}
+
+{% anchor h3 %}
+Send transactional email using the SendGrid API
+{% endanchor %}
+
+*To send a dynamic transactional email:*
+
+In order to send a dynamic transactional email using cURL, set your call up to look something like this:
+
+{% codeblock %}
+curl -X "POST" "https://api.sendgrid.com/v3/mail/send" \
+     -H 'Authorization: Bearer [YourAPIKeyGoesHere!]' \
+     -H 'Content-Type: application/json' \
+     -d $'{
+  "from": {
+    "email": "example@.sendgrid.net"
+  },
+  "personalizations": [{
+    "to": [{
+      "email": "example@sendgrid.net"
+    }],
+    "dynamic_template_data": {
+      "guest": "Anthony Wang"
+      "partysize": 4,
+      "english": true,
+      "date": "April 1st, 2018"
+    }
+  }],
+  "template_id": "[template_id]",
+}'
+{% endcodeblock %}
+
+It is important to note two sections of this call when using dynamic templates.
+
+In order to send dynamic content, you need to specify a collection of key/value pairs in the `dynamic_template_data` object following the pattern `"handlebar_tag":"value to substitute"`. All are assumed to be strings. These handlebar substitutions apply to the text and html content of the body of your email, in addition to the subject and reply-to parameters. The total collective size of your handle bar substitutions may not exceed 10,000 bytes per personalization object.
+
+{% codeblock %}
+ }],
+    "dynamic_template_data": {
+      "guest": "Anthony Wang"
+      "partysize": 4,
+      "english": true,
+      "date": "April 1st, 2018"
+    }
+  }],
+{% endcodeblock %}
+
+In addition to specifying the dynamic template data, you need to specify the template ID. The template ID is 64 characters with one dash (d-uuid). If you forget your template ID and want to access it from the API, use the following curl call to retrieve all of your transactional templates:
+
+{% codeblock %}
+curl --request GET \
+  --url 'https://api.sendgrid.com/v3/templates?generations\[\]=legacy&generations\[\]=dynamic' \
+  --header 'Authorization: Bearer SG.keyhere' \
+  --header 'Content-Type: application/json'
+{% endcodeblock %}
+
+You can also copy your template ID from the app and paste it into your call. 
+
+![]({{root_url}}/img/dynamic_template_id.png "Dynamic Template ID")
+
+{% anchor h3 %}
+Send transactional email using SMTP
+{% endanchor %}
+
+Coming soon!
+
+{% anchor h2 %}
+Additional Resources
+{% endanchor h2 %}
+
```

Comment:
add additional resource for the using handlebars page.

Might be good to go through all of these and make sure it's all crosslinked well

Reply:
done.